server/storage/endpoint/key_path.go

```diff
@@ -103,3 +104,23 @@ func gcSafePointServicePath(serviceID string) string {
 func MinResolvedTSPath() string {
 	return path.Join(clusterPath, minResolvedTS)
 }
+
+// gcSafePointPathByServiceGroup returns the path of the gc_worker's safe point
+// /gc_servicegroup/gc_safepoint/$service_group_id
+func gcSafePointPathByServiceGroup(serviceGroupID string) string {
+	return path.Join(gcServiceGroupPath, "gc_safepoint", serviceGroupID)
```

Comment:
suggest to define the path including `gc_safepoint ` `service_safepoint` as const.